### DISCUSSION TITLE
Editing document in Cloud Studio expand document doesn't update on refresh - bug?

### DISCUSSION THREAD

**Discussion by genachka** [ravendb/ravendb#14017] [General]
Using Cloud Studio v5.3.101, when editing a document, setting it to `Collapse document` and if that document is updated from somewhere else (i.e. an app), the "Click here to refresh" appears in the Studio document editor. When you click on "Click here to refresh" the document in the editor is displayed refreshed and is shown expanded but the button still says `Expand document` instead of `Collapse document` and requires clicking twice on it to collapse it again the document.

**Comment by ml054** [comment]
Hi, @genachka 

I've moved this to our official bugtracker. You can track progress here: https://issues.hibernatingrhinos.com/issue/RavenDB-18441

**Reply by genachka** [reply]
@ml054 thank you. One more issue I noticed if you can add to that report that's related...

If you collapse the contents, then click on the `Format` button, when it re-formats everything it expands all the sections that were collapsed but the button still says `Expand` when it should have reverted to `Collapse`

**Reply by ml054** [reply]
@genachka Reported - thanks :)
